spec/Data_Parallelism.tex

```diff
@@ -52,7 +52,10 @@ \subsection{Syntax}
 \end{syntax}
 As with the for statement, the indices may be omitted if they are
 unnecessary and the \chpl{do} keyword may be omitted before a block
-statement.  The square bracketed form is a syntactic convenience.
+statement.  The square bracketed form will resort to serial iteration
```

Comment:
I would make this a separate paragraph in both places.  I might still mention that the square bracketed form is for convenience, while still emphasizing its difference in behavior.